index.js

```diff
@@ -158,10 +174,53 @@ const ContainerService = {
     }).catch(() => {
       throw new Error("WebExtension startup failed. Unable to continue.");
     });
+
+    this._sendEvent = new Metrics({
+      type: "sdk",
+      id: self.id,
+      version: self.version
+    }).sendEvent;
+
   },
 
   // utility methods
 
+  _containerTabCount(userContextId) {
```

Comment:
Just to make this code easy to follow, add a comment: "This method returns the total of open and hidden tabs with this userContextId" or something similar...